Sources/StytchCore/StytchB2BClient/StytchB2BClient+Passwords.swift

```diff
@@ -58,7 +58,7 @@ public extension StytchB2BClient {
         /// Reset the member’s password and authenticate them. This endpoint checks that the existing password matches the stored value.
         ///
         /// The provided password needs to meet our password strength requirements, which can be checked in advance with the password strength endpoint. If the password and accompanying parameters are accepted, the password is securely stored for future authentication and the member is authenticated.
-        public func resetByExistingPassword(parameters: ResetByExistingPasswordParameters) async throws -> B2BAuthenticateResponse {
+        public func resetByExistingPassword(parameters: ResetByExistingPasswordParameters) async throws -> B2BMFAAuthenticateResponse {
```

Comment:
Marking this here, since GH won't let me add a comment where it goes, but Passwords.resetByEmail and Passwords.authenticate take ISTs, too

Reply:
fixed